src/types.ts

```diff
@@ -17,9 +17,17 @@ export type SignedMessage = {
 };
 
 export type WallyConnectorOptions = {
+  // The clientId you get when signing up
   clientId: string;
+  // Required when using devUrl or token
   isDevelopment?: boolean;
+  // The local instance of the wally api. APP ROOT used if unset or not dev mode
   devUrl?: string;
+  // If you're testing stuff out in dev mode and don't have access to the
+  // login flow (i.e. chrome extension), but you do have a token, put it here
+  token?: string;
```

Comment:
what do you mean by token?

Reply:
from huddle chat: will probably go a different direction with this & the chrome extension - likely a dynamic redirect url which would let us redirect to anywhere in the context of the extension